spring-aot/src/main/java/org/springframework/context/bootstrap/generator/nativex/HintsBeanNativeConfigurationProcessor.java

```diff
@@ -44,16 +50,12 @@
 import org.springframework.nativex.type.TypeSystem;
 import org.springframework.util.ClassUtils;
 
-import static org.springframework.context.bootstrap.generator.infrastructure.nativex.NativeConfigurationRegistry.InitializationConfiguration;
```

Comment:
I know we don't have a consistent code style at the moment but can you please re-read the commit to avoid such unrelated changes?